src/js/EpubReader.js

```diff
@@ -733,38 +733,49 @@ BookmarkData){
             $("#right-page-btn").show();
         else
             $("#right-page-btn").hide();
-    }
+    };
+
+    var generateQueryParamCFI = function (bookmark) {
+        if (!bookmark.idref) {
+            return;
```

Comment:
My only reservation for this Pull Request: this return statement yields an `undefined` value instead of a string, which means that the `goto` property on the caller side of `generateQueryParamCFI()` will trickle down into the `overrideEntry` value in the `Helpers.buildUrlQueryParameters()` function of `readium-shared-js`.

Is there a better way? Perhaps if result of `generateQueryParamCFI()` is undefined, then `goto: " "` (space character) to be consistent with other usages with `epubs` and `embedded` (which means "do not override this URL querystring param, use the existing `window.location` value").

Reply:
@danielweck I see what you mean. I'll correct this.